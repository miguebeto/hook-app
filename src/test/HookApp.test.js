import React from 'react';
import { shallow } from 'enzyme';
import { HookApp } from '../HookApp';

describe('Pruebas en el componente <HookApp />', ()=> {


    test('Debe mantenerse el componente sin cambios en el renderizado', ()=>{

        //debe retornar el componente sin cambios en el render
        const wrapper = shallow(<HookApp />);
        expect(wrapper).toMatchSnapshot();
    });
});