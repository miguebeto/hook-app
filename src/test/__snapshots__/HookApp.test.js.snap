// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente <HookApp /> Debe mantenerse el componente sin cambios en el renderizado 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
